integration/e2ecortex/client.go

```diff
@@ -363,6 +329,40 @@ type userConfig struct {
 	AlertmanagerConfig string            `yaml:"alertmanager_config"`
 }
 
+// GetAlertmanagerConfig gets the status of an alertmanager instance
+func (c *Client) GetAlertmanagerConfig(ctx context.Context) (*alertConfig.Config, error) {
```

Comment:
Any reason why was moved? Not that I'm against code re-organisation, but it makes diff harder to read. I tried to compare it with `master` and I've the feeling nothing has changed, but maybe I'm missing something.

Reply:
Sorry! I'm happy to move it back if needbe - nothing changed, this was grouped with the functions that related to the ruler so I moved it further down to be with the functions that relate to the Alertmanager.